### PR TITLE
Straw hat

### PR DESCRIPTION
**Pull request by Adrian16199**
Farmer's best friend. (totally)

TO-DO: Make it craftable to be wheat, probably need a special tag.... eh= Done

We did it boys.

## About the PR

Hat
## Why / Balance

Botany deserves love.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image
image

**Changelog**

Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Crew has learned how to make a straw hat out of wheat bushels!

### CODE REVIEW

**Comment by Colin-Tel** [comment]
YAML linter is failing. Is it because you put hyphens before “tag” in the crafting graph?

Also your changelog won’t get added to the changelog because it’s formatted improperly. Was this intentional?

**Comment by Adrian16199** [comment]
Im pretty sure I fixed the changelog but I have no idea what is wrong with the yaml linter. I checked every single file and compared it to diferent files, only thing that I can imagine being wrong is either one too many spaces or one too little.

**Comment by deltanedas** [comment]
learned not learnt :trollface:

add a space between `-tag` so its `- tag`

**Comment by Adrian16199** [comment]
I sure do love spacing. Ingame and in coding.

File: Resources/Prototypes/DeltaV/Recipes/Crafting/improvised.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- type: construction
+  name: Straw hat
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Recipes/Crafting/improvised.yml]:
```suggestion
  name: straw hat
```

File: Resources/Prototypes/DeltaV/Entities/Clothing/Head/hats.yml [hunk 2]
```diff
@@ -192,3 +192,21 @@
     sprite: DeltaV/Clothing/Head/Hats/beret_corpsman.rsi
   - type: Clothing
     sprite: DeltaV/Clothing/Head/Hats/beret_corpsman.rsi
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatStrawHat
+  name: Straw hat
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Head/hats.yml]:
```suggestion
  name: straw hat
```

File: Resources/Prototypes/DeltaV/Entities/Clothing/Head/hats.yml [hunk 3]
```diff
@@ -192,3 +192,21 @@
...
+  description: A fancy hat for hot days, not recommended next to fires.
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Head/hats.yml]:
```suggestion
  description: A fancy hat for hot days! Not recommended to wear near fires.
```
I’m on phone so I can’t do a multi line review comment 😭

File: Resources/Prototypes/DeltaV/Recipes/Crafting/improvised.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  id: strawHat
+  graph: StrawHat
+  startNode: start
+  targetNode: strawhat
+  category: construction-category-clothing
+  description: A fancy hat for hot days, not recommended next to fires.
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Recipes/Crafting/improvised.yml]:
```suggestion
  description: A fancy hat for hot days! Not recommended to wear near fires.
```

**Review by Colin-Tel — CHANGES_REQUESTED** [review]
Some grammar/capitalization to address

**Comment by Adrian16199** [comment]
Still works after changes so definitely nice.

**Review by Colin-Tel — APPROVED** [review]

**Pull request merged**